pyfr/plugins/turbulence.py

```diff
@@ -0,0 +1,256 @@
+import math
+import numpy as np
+
+from collections import defaultdict
+from pyfr.plugins.base import BaseSolverPlugin
+from pyfr.regions import BoxRegion
+from rtree.index import Index, Property
+
+# pcg32rxs_m_xs
+def pcg(seed):
+    state = np.uint32(seed)
+    multiplier = np.uint32(747796405)
+    increment = np.uint32(2891336453)
+    mcgmultiplier = np.uint32(277803737)
+    n4 = np.uint32(4)
```

Comment:
Can move these onto the same line `n4, n22, n28 = np.uint32(4), np.uint32(22), np.uint32(28)`